docs/index.rst

```diff
@@ -2,42 +2,64 @@
 sunpy core Documentation
 ************************
 
-The sunpy core package is a community-developed, free and open-source solar
+``sunpy`` is a community-developed, free and open-source solar
 data analysis environment for Python.
 
+New Users
+*********
+
+If you are a first time user or new to Python,
+
+..
+        * :Introduction to sunpy core [new page described in #6615]
```

Comment:
There is actually a sphinx extension which supports todo items and tracking them, might be worth a look: https://www.sphinx-doc.org/en/master/usage/extensions/todo.html ?